### PR TITLE
Lint APPSODY_DEBUG_PORT environment variable

### PR DESCRIPTION
**Pull request by Kamran64**
Adds the `APPSODY_DEBUG_PORT` variable to the linter as a warning.

### CODE REVIEW

File: cmd/stack_lint_dockerfile_stack.go [hunk 1]
```diff
@@ -79,8 +79,12 @@ func lintDockerFileStack(log *LoggingConfig, stackPath string) (int, int) {
 			}
 		}
 		if !variableFound {
-			log.Warning.log("Missing ", variable)
-			stackLintWarningCount++
+			if optionalEnvironmentVariables[i] == "APPSODY_DEBUG_PORT" {
+				log.Warning.log("Missing APPSODY_DEBUG_PORT. If your stack supports the appsody debug command, please define this variable.")
+			} else {
+				log.Warning.log("Missing ", variable)
+				stackLintWarningCount++
```
Comment (EnriqueL8) [inline cmd/stack_lint_dockerfile_stack.go:86]: you may want to increment the counter in both cases?

File: cmd/stack_lint_dockerfile_stack.go [hunk 2]
```diff
@@ -79,8 +79,12 @@ func lintDockerFileStack(log *LoggingConfig, stackPath string) (int, int) {
 			}
 		}
 		if !variableFound {
-			log.Warning.log("Missing ", variable)
-			stackLintWarningCount++
+			if optionalEnvironmentVariables[i] == "APPSODY_DEBUG_PORT" {
+				log.Warning.log("Missing APPSODY_DEBUG_PORT. If your stack supports the appsody debug command, please define this variable.")
```
Comment (EnriqueL8) [inline cmd/stack_lint_dockerfile_stack.go:83]: You could check for `APPSODY_DEBUG` to be set? And only through the error if so? you could also concatinate the errmg and pass it to `log.Warning.log("Missing ", variable, errMsgDebug)`

Comment (Kamran64) [reply cmd/stack_lint_dockerfile_stack.go]: This sounds like a better way of doing it, done 👍

File: cmd/stack_lint_dockerfile_stack.go [hunk 3]
```diff
@@ -77,9 +78,16 @@ func lintDockerFileStack(log *LoggingConfig, stackPath string) (int, int) {
...
 				variableFound = true
 			}
+			if k == "APPSODY_DEBUG" {
+				appsodyDebugFound = true
+			}
 		}
 		if !variableFound {
-			log.Warning.log("Missing ", variable)
+			errMsg := ""
+			if appsodyDebugFound == true && optionalEnvironmentVariables[i] == "APPSODY_DEBUG_PORT" {
```
Comment (EnriqueL8) [inline cmd/stack_lint_dockerfile_stack.go:87]: you can just do `if appsodyDebugFound` instead of `== true`, sorry for the nitpick

Comment (Kamran64) [reply cmd/stack_lint_dockerfile_stack.go]: Np, i think the go linter fails (the irony) with that anyway

**Pull request merged**